plans/smlbench2/utils/files.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"bufio"
+	"crypto/rand"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/ipfs/testground/sdk/runtime"
+)
+
+// TempRandFile creates a file of the specified size (in bytes) within the
+// specified directory.
+//
+// It is the callers responsibility to delete this file when done.
+func TempRandFile(runenv *runtime.RunEnv, dir string, size int64) *os.File {
```

Comment:
This util will probably end up on the sdk as I'm sure it will be used many times across many plans :)

Reply:
Once we write a few more tests, I think it's going to become clear what needs to go into the SDK. Already, I can see a lot of things that don't have to be boilerplate.